book/02-using-atom/sections/01-packages-themes.asc

```diff
@@ -1,7 +1,7 @@
 [[_atom_packages]]
 === Atom Packages
 
-First we'll start with the Atom package system. As we mentioned previously, Atom itself is a very basic core of functionality that ships with a number of useful packages that add new features like the Tree view and the Settings view (which, by the way, can be found here: https://github.com/atom/settings-view).
+First we'll start with the Atom package system. As we mentioned previously, Atom itself is a very basic core of functionality that ships with a number of useful packages that add new features like the Tree view and the Settings view (which, by the way, can be found here: https://github.com/atom/settings-view[https://github.com/atom/settings-view]).
```

Comment:
I think it makes sense to use the CapWords convention for the package title, like `url[Tree View]` or `url[Settings View]`, since they also use that convention on their respective repo READMEs, e.g. in https://github.com/atom/settings-view.

Do you agree with that?